src/components/RewardWrapper/index.tsx

```diff
@@ -344,11 +349,12 @@ export function RewardWrapper({
     if (account.address && connectionHasSwitchChain) {
```

Comment:
shouldn't we move `&& connectionHasSwitchChain` out of this condition?

Reply:
Yes I think so, but didn't want to expand scope on this PR. I can make the change though